docs/src/tutorial-first-steps.md

```diff
@@ -63,11 +61,10 @@ To see traces, run:
 
 ```
 kani --visualize src/lib.rs
-open report/html/index.html
 ```
 
-The first command runs Kani and generates the html-based report in `report/`.
-The second command opens that report in your default browser (on mac, on linux desktops try `xdg-open`).
+This command runs Kani and generates the HTML report in `report/html/index.html`.
+Open the report with your favorite browser.
```

Comment:
"preferred browser" might be a better way to phrase this.

Reply:
Done.